utils/soca/gdas_ens_handler.h

```diff
@@ -0,0 +1,263 @@
+#pragma once
+
+#include <netcdf>
```

Comment:
what is netCDF used for?

Reply:
good catch ... not used.

Reply:
removed netcdf in one of the latest commits